apps/web/components/booking/BookingListItem.tsx

```diff
@@ -322,7 +323,8 @@ function BookingListItem(booking: BookingItemProps) {
                 "max-w-56 truncate text-sm font-medium leading-6 text-neutral-900 md:max-w-max",
                 isCancelled ? "line-through" : ""
               )}>
-              {booking.eventType?.team && <strong>{booking.eventType.team.name}: </strong>}
+              {booking.eventType?.team && <Badge variant="gray">{booking.eventType.team.name}</Badge>}
+              <span> </span>
               {booking.title}
```

Comment:
Since the team owner may not be the actual attendee, can we add the actual attender here? cc @PeerRich and @Jaibles for more product feedback

Reply:
Yes we should have the host & attendees displayed for sure.
<img width="891" alt="CleanShot 2022-08-31 at 11 23 47@2x" src="https://user-images.githubusercontent.com/4536123/187657344-22e4d56f-c7da-4198-a98e-04f9c2098540.png">

In designs I've actually added it so it would always show "You" anyway, so makes sense to have that and then swap it with the host in this instance.

Reply:
should this be a new PR or in this PR? @zomars @Jaibles

Reply:
@PeerRich yes seems like a new addition, I think this PR should be good to go out on it's own and then a subsequent PR can be done to improve who is listed.

Reply:
My call is this one, since not having this will cause confusion on what are you actual bookings

Reply:
<img width="1152" alt="Screenshot 2022-09-02 at 5 45 15 PM" src="https://user-images.githubusercontent.com/53316345/188140786-1b86d7cc-55f3-4570-9bb3-a0faa9371aa8.png">

ok so currently it shows the email of the person (below the title) who has booked the meeting. you want to display usernames of all participants of the meeting ?

Reply:
i think that is good to merge and then we do a follow up PR after 2.0